examples/address_generator/src/main.rs

```diff
@@ -0,0 +1,134 @@
+//
+// Copyright 2019 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::{App, Arg};
+
+use transact::contract::address::{
+    double_key_hash::DoubleKeyHashAddresser, key_hash::KeyHashAddresser,
+    triple_key_hash::TripleKeyHashAddresser, Addresser,
+};
+
+fn main() -> Result<(), String> {
+    let matches = App::new(clap::crate_name!())
+        .version(clap::crate_version!())
+        .author(clap::crate_authors!())
+        .about(clap::crate_description!())
+        .arg(
+            Arg::with_name("prefix")
+                .long("prefix")
+                .takes_value(true)
+                .required(true)
+                .help("the prefix of the radix address"),
+        )
+        .arg(
+            Arg::with_name("keys")
+                .long("keys")
+                .required(true)
+                .takes_value(true)
+                .max_values(3)
+                .help(
+                    "the natural keys used to compute the radix address, \
+                     must supply 1 - 3 separate keys",
+                ),
+        )
+        .arg(
+            Arg::with_name("lengths")
+                .long("lengths")
+                .required(false)
+                .takes_value(true)
+                .max_values(2)
+                .help(
+                    "optional lengths when multiple keys are used to compute the radix address \
```

Comment:
This should start with a capital letter. Try and make it work such that when you have a terminal of with 80 it looks ok.